models/category.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const categorySchema = new mongoose.Schema({
+    categoryName: {
```

Comment:
since you are in the category, rename `categoryName` to `name`
```suggestion
    name: {
```